### PR TITLE
Adding custom icons used in the Ignite UI product

### LINKED ISSUE #6: Add icons
- [x] add a row in a grid
- [x] add a child row under this parent in the tree grid

### LINKED ISSUE #8: Create a new category and add all custom Ignite UI icons to it
Create a new category and add all custom Ignite UI icons to it such as: pinning and filter conditions etc.

### PR DESCRIPTION
**Pull request by StefanIvanov**
Closes #6 
Closes #8

### CODE REVIEW

File: src/editor.ts [hunk 1]
```diff
@@ -0,0 +1,216 @@
+import { IMXIcon } from "./common/icon";
+import { unpinLeft, unpinRight } from "./content";
```
Comment (simeonoff) [inline src/editor.ts]: What's the point of this unused import? There's a reason why `categories` is an array. A single icon can have more than one category in its list of categories.
```suggestion
```

File: src/svgs/pin-right.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
+<path d="M8.9,2L1.8,9.1l2.8,0l3.5,3.6l0,4.2l3.5-3.5l5.7,5.7h1.4v-1.4l-5.7-5.7l3.5-3.5l-4.2,0L8.9,4.9C8.9,4.9,8.9,2,8.9,2z"/>
+</svg>
```
Comment (simeonoff) [inline src/svgs/pin-right.svg]: This should be simplified.

File: src/svgs/unpin-right.svg [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<!-- Generator: Adobe Illustrator 24.2.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:none;}
+</style>
+<path class="st0" d="M0,0h24v25H0V0z"/>
+<path d="M12.4,14.1l5.1,5.1h1.4v-1.4l-5.1-5.1l8.4-8.4L20.9,3l-18,18l1.3,1.2L12.4,14.1z M8.4,13.6l5-5h-0.7L9.1,5V2.2L2,9.3h2.8
+	l3.5,3.5L8.4,13.6z"/>
+</svg>
```
Comment (simeonoff) [inline src/svgs/unpin-right.svg]: This should be simplified.

**Review by simeonoff — CHANGES_REQUESTED** [review]
Most SVGs are badly formatted ans some need further simplification. See my comments.

File: src/editor.ts (hunk 1 above)
Comment (StefanIvanov) [reply src/editor.ts]: I typed them by mistake and then removed them but didn't notice an import was added at the top, thanks for noticing

File: src/svgs/pin-right.svg (hunk 2 above)
Comment (StefanIvanov) [reply src/svgs/pin-right.svg]: yes

File: src/svgs/unpin-right.svg (hunk 3 above)
Comment (StefanIvanov) [reply src/svgs/unpin-right.svg]: yes

**Review by simeonoff — APPROVED** [review]

**Pull request merged**